DotaStats/Modules/MatchInfoModule/View/AdditionalMatchInfoTableViewCell.swift

```diff
@@ -0,0 +1,140 @@
+import UIKit
+
+final class AdditionalMatchInfoTableViewCell: UITableViewCell {
+    static let reuseIdentifier = "AdditionalMatchInfoTableViewCell"
+    var matchID = String()
+
+    private lazy var regionStackView: UIStackView = {
+        let regionStackView = UIStackView()
+        createStackView(stackView: regionStackView, axis: .vertical, spacing: 8)
+        return regionStackView
+    }()
+
+    private lazy var skillStackView: UIStackView = {
+        let skillStackView = UIStackView()
+        createStackView(stackView: skillStackView, axis: .vertical, spacing: 8)
+        return skillStackView
+    }()
+
+    private lazy var matchIdCopyButton: CopyIdButton = {
+        let matchIdCopyButton = CopyIdButton()
+        matchIdCopyButton.setTitle("Copy match ID", for: .normal)
+        matchIdCopyButton.layer.cornerRadius = 15
+        matchIdCopyButton.backgroundColor = ColorPalette.alternativeBackground
+        matchIdCopyButton.setTitleColor(ColorPalette.text, for: .normal)
+        matchIdCopyButton.titleLabel?.font = UIFont.systemFont(ofSize: 20)
+        matchIdCopyButton.addTarget(self, action: #selector(copyMatchIdButtonPressed), for: .touchUpInside)
+        return matchIdCopyButton
+    }()
+
+    private lazy var regionNameLabel: UILabel = {
+        let regionNameLabel = UILabel()
+        regionNameLabel.text = "REGION"
+        regionNameLabel.textColor = ColorPalette.subtitle
+        regionNameLabel.font = UIFont.systemFont(ofSize: 15)
+        return regionNameLabel
+    }()
+
+    private lazy var regionLabel: UILabel = {
+        let regionLabel = UILabel()
+        regionLabel.textColor = ColorPalette.mainText
+        regionLabel.font = UIFont.systemFont(ofSize: 15)
+        regionLabel.numberOfLines = 3
+        regionLabel.textAlignment = .center
+        return regionLabel
+    }()
+
+    private lazy var skillNameLabel: UILabel = {
+        let skillNameLabel = UILabel()
+        skillNameLabel.text = "SKILL"
+        skillNameLabel.textColor = ColorPalette.subtitle
+        skillNameLabel.font = UIFont.systemFont(ofSize: 15)
+        return skillNameLabel
+    }()
+
+    private lazy var skillLabel: UILabel = {
+        let skillLabel = UILabel()
+        skillLabel.textColor = ColorPalette.mainText
+        skillLabel.font = UIFont.systemFont(ofSize: 15)
+        skillLabel.numberOfLines = 3
+        skillLabel.textAlignment = .center
+        return skillLabel
+    }()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setup()
+        setUpConstraints()
```

Comment:
Пжлст, давай `setup` писать однообразно. Посмотри по всему коду
```suggestion
        setupConstraints()
```

Reply:
сделала